rpc/tests/test_seth_rpc.py

```diff
@@ -1065,15 +1068,17 @@ def test_get_log_filter_changes(self):
             )])],
         ))
 
+        receipt = SethTransactionReceipt(
+                gas_used=self.gas,
+                return_value=self.return_value_b,
+                contract_address=self.contract_address_b,
+                status=self.status,
+                to=self.contract_address_b,
+                )
+        setattr(receipt, 'from', self.account_address_b)
```

Comment:
Wondering what the purpose of `setattr` is here vs just passing it into the constructor

Reply:
Sadly due to a limitation of python you can't use reserved words as keyword arguments for a function. The protobuf devs [seem to be aware of that limitation, but it's as yet unresolved](https://github.com/protocolbuffers/protobuf/blob/master/python/google/protobuf/internal/python_message.py#L220-L236), and `setattr` is the only viable workaround I could find.

Reply:
I'll add a comment to make this more clear, since I'm already going to do that `bytes.fromhex` fix you mentioned elsewhere...

Reply:
Cool, makes sense. Another option is that you could do is `SethTransactionReceipt(**{'from': self.account_address_b, ...})`, but either way works